src/modules/people-list/people-list-item.tsx

```diff
@@ -0,0 +1,203 @@
+        ? 'Female'
+        : 'Male';
+
+    if (strGender !== undefined && strGender.length > 0) {
+      arrOtherDet.push(strGender);
+    }
+
+    if (marStatus !== undefined && marStatus.length > 0) {
+      arrOtherDet.push(marStatus);
+    }
+
+    if (churchEntName !== undefined && churchEntName.length > 0) {
+      arrOtherDet.push(churchEntName);
+    }
+
+    return arrOtherDet;
+  };
+
+  const renderUserPic = () => {
+    return (
+      <StyleUserProfilePicWrapper
+        uri={profilePic!}
+        initials={getUserNameFirstLastCharacter()}
+        size={40}
+        radius="small"
+        // testID="profile-pic"
+      />
+    );
+  };
+
+  const getUserNameFirstLastCharacter = () => {
+    const arrName = name?.split(' ');
+    if (arrName?.length === 2) {
+      return `${arrName[0].charAt(0)}${arrName[1].charAt(0)}`;
+    }
+    return `${arrName![0].charAt(0)}${arrName![0].charAt(1)}`;
+  };
+
+  const renderCheckBoxOrPic = () => {
+    if (isShowCheckbox && isShowCheckbox === true) {
+      return (
+        <StyleProfilePicCheckboxWrapper>
+          <StyleCheckboxWrapper>
+            <Checkbox onPress={onCheckboxPress} isChecked={isChecked!} />
+          </StyleCheckboxWrapper>
+          {renderUserPic()}
+        </StyleProfilePicCheckboxWrapper>
+      );
+    }
+    return renderUserPic();
+  };
+
+  return (
+    <Swipeable
+      leftButtonWidth={minorScale(17)}
+      leftButtons={leftButtons}
+      rightButtonWidth={minorScale(17)}
+      rightButtons={rightButtons}
+      leftActionActivationDistance={0}>
+      <StyleLinkWrapper testID="link-wrapper" onPress={() => onUserClicked?.()}>
+        <StyleWrapper testID="main-wrapper">
+          {/* User Profile Image and checkbox */}
+          {renderCheckBoxOrPic()}
+          {/* End User Profile Image and checkbox */}
+          {/* user details View */}
+          <StyleUserDetailsWrapper isShowCheckbox={isShowCheckbox}>
+            {/* user name and User id */}
+            <StyleUserNameIdWrapper>
+              {/* user name */}
+              <StyleHeadingUserNameWrapper variant="h3">{`${name}`}</StyleHeadingUserNameWrapper>
+              {/* User Id */}
+              <StyleUserIdTextWrapper testID="user-id">
+                ID:{`${userId}`}
+              </StyleUserIdTextWrapper>
+            </StyleUserNameIdWrapper>
+            {/* End user name and User id */}
+
+            {/* footer element and other user details */}
+            <StyleOtherDetailWrapper>
+              {/* user other information like gender,maritalStatus and churchEntityName */}
+              <StyleUserDescriptionWrapper>
+                {`${getOtherDetailsString().join(' | ')}`}
+              </StyleUserDescriptionWrapper>
+              {/* footer element */}
+              {footerElement}
+            </StyleOtherDetailWrapper>
+            {/* End footer element and other user details */}
+          </StyleUserDetailsWrapper>
+        </StyleWrapper>
```

Comment:
Can we please remove these comments? One of the benefits of using Styled Components is that we can easily understand what a tag (eg `StyledUserNameIdWrapper` or `StyleOtherDetailWrapper`) is doing.

Reply:
i have remove all comments